app/models/review_questionnaire.rb

```diff
@@ -21,8 +21,13 @@ def get_assessments_round_for(participant, round)
     responses = []
     if participant
       maps = ResponseMap.where(reviewee_id: team_id, type: "ReviewResponseMap")
+      # puts("Maps +++++++")
+      # puts(maps)
+      puts "maps response"
```

Comment:
Again, you should remove debug code. Please do not comment out debug code, remove them.